content/self-host/rustdesk-server-pro/smtp/Microsoft365/_index.zh-cn.md

```
@@ -22,7 +22,7 @@ keywords: ["rustdesk smtp oauth2", "rustdesk smtp microsoft 365", "rustdesk smtp
 | OAuth2 Client secret | `Certificates & secrets` 中创建的 secret `Value` |
 
 下图可帮助你直接对应 RustDesk 中这些字段的位置：
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)